chain-connect/src/chainApis/TokenApi.ts

```diff
@@ -13,45 +13,46 @@
  * limitations under the License.
  */
 import {
-  BatchMintTokenParams,
-  BurnTokensParams,
-  CreateTokenClassParams,
-  DeleteAllowancesParams,
+  BatchMintTokenDto,
+  BurnTokensDto,
+  CreateTokenClassDto,
+  DeleteAllowancesDto,
   FetchAllowancesBody,
-  FetchAllowancesParams,
-  FetchBalancesParams,
-  FetchBalancesWithPaginationParams,
+  FetchAllowancesDto,
+  FetchBalancesDto,
+  FetchBalancesWithPaginationDto,
   FetchBalancesWithTokenMetadataBody,
-  FetchBurnsParams,
-  FetchMintRequestsParams,
-  FetchTokenClassesParams,
+  FetchBurnsDto,
+  FetchMintRequestsDto,
+  FetchTokenClassesDto,
   FetchTokenClassesResponseBody,
-  FetchTokenClassesWithPaginationParams,
+  FetchTokenClassesWithPaginationDto,
   FulfillMintDto,
-  FulfillMintParams,
-  FullAllowanceCheckParams,
+  FullAllowanceCheckDto,
+  FullAllowanceCheckResDto,
   GalaChainResponse,
-  GrantAllowanceParams,
-  HighThroughputMintTokenParams,
-  LockTokenRequestParams,
-  LockTokensParams,
+  GrantAllowanceDto,
+  HighThroughputMintTokenDto,
+  LockTokenDto,
+  LockTokensDto,
   MintRequestDto,
-  MintTokenParams,
-  MintTokenWithAllowanceParams,
-  RefreshAllowanceParams,
-  ReleaseTokenParams,
+  MintTokenDto,
+  MintTokenWithAllowanceDto,
+  RefreshAllowanceDto,
+  ReleaseTokenDto,
   TokenAllowanceBody,
+  TokenBalance,
   TokenBalanceBody,
   TokenBurnBody,
```

Comment:
Are the body types still needed? I think those were also constructorArg wrapped types?